src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/GameReports/StandingsEntryViewModel.cs

```diff
@@ -155,11 +150,13 @@ public static List<StandingsEntryViewModel> SetPositions(List<StandingsEntryView
         /// <param name="firstEntry">First team standings entry</param>
         /// <param name="secondEntry">Second team standings entry</param>
         /// <returns>True if teams have same position</returns>
+#pragma warning disable S1244 // Want to check floating point not with exact values, but with range
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)